sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -50,6 +62,9 @@
     private String testName;
     private final Scheduler scheduler = Schedulers.parallel();
 
+    static final byte[] CONTENTS_BYTES = "Some-contents".getBytes(StandardCharsets.UTF_8);
+    String sessionId;
```

Comment:
private modifier for both.

Reply:
sessionId : This will be needed in `ServiceBusReceiverClientIntegrationTest`  to create receiver and sender.